assets/src/util/assignment.js

```diff
@@ -1,5 +1,5 @@
 import { calculateWeekOffset, dateToMonthDay } from './date'
-import { sum, roundToXDecimals, getDecimalPlaceOfFloat } from './math'
```

Comment:
You may still want this function to help with what I described in the other comment, but if you don't end up using it, we should delete the definition in `math.js`.

Reply:
I ended up using it to fix your prior review comment.